### PR TITLE
Modify Dockerfile conda to conda-forge

### PR DESCRIPTION
**Pull request by WeizhuoZhang-intel**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -61,6 +59,7 @@ RUN curl -fsSL -v -k -o ~/miniconda.sh -O https://repo.anaconda.com/miniconda/Mi
         openssl \      
         weasyprint && \
     /opt/conda/bin/conda clean -ya && \
+    pip install intel-openmp && \
```
Comment (chuanqi129) [inline docker/Dockerfile:62]: Why we need this change?

**Review by chuanqi129 — COMMENTED** [review]
Please apply this change to all Dockerfiles

File: docker/Dockerfile (hunk 1 above)
Comment (WeizhuoZhang-intel) [reply docker/Dockerfile:62]: Conda-forge uses `conda-forge` channel by default, which has no `intel-openmp` pkg. If we want to install `intel-openmp` through `conda`, it only can be installed through intel's conda channel by indicating `conda install intel-openmp -c intel `. I am not that we were able to use intel channel or not. I am checking this issue with related colleage.

Comment (chuanqi129) [reply docker/Dockerfile:62]: Yes, we can use intel channel

File: docker/Dockerfile [hunk 2]
```diff
@@ -27,12 +27,11 @@ ENV PATH /opt/conda/bin:$PATH
...
+RUN curl -fsSL -v -k -o ~/miniforge.sh -O https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh && \
+    chmod +x ~/miniforge.sh && \
     # Fix conda install issue
-    sed -i "s;#!/bin/sh;#!/bin/bash;" ~/miniconda.sh && \
-    ~/miniconda.sh -b -p /opt/conda && \
-    rm ~/miniconda.sh && \
+    ~/miniforge.sh -b -p /opt/conda && \
+    rm ~/miniforge.sh && \
     /opt/conda/bin/conda config --set channel_priority strict && \
     /opt/conda/bin/conda config --append channels conda-forge && \
```
Comment (chuanqi129) [inline docker/Dockerfile:36]:
```suggestion
    /opt/conda/bin/conda config --append channels intel && \
```

**Comment by WeizhuoZhang-intel** [comment]
> Please apply this change to all Dockerfiles

Done.

File: docker/Dockerfile (hunk 1 above)
Comment (WeizhuoZhang-intel) [reply docker/Dockerfile]: Resolved

**Review by chuanqi129 — APPROVED** [review]
We'd better to launch a basic config test based on this change to check whether if have performance impact

**Pull request merged**